### DISCUSSION TITLE
Connect smplayer to external mpv

### DISCUSSION THREAD

**Discussion by bt4ibwem8** [smplayer-dev/smplayer#975] [General]
Is it possible to use my already installed mpv in Windows system instead of built-in mpv into smplayer? So i could use the latest version of mpv with my settings together with smplayer...

**Comment by smplayer-dev** [comment]
Yes, set the mpv path in preferences -> general.

**Reply by bt4ibwem8** [reply]
Great, i somehow missed that setting. Thanks.

**Comment by bt4ibwem8** [comment]
Can i also map/bind smplayer to my `portable_config` folder inside mpv installation?
